distribution/docs/src/main/jdocs/content/_security-framework/security-encryption-commands-contents.adoc

```diff
@@ -0,0 +1,31 @@
+:title: Security Encryption Commands
+:type: subSecurityFramework
+:status: published
+:parent: Security Encryption
+:order: 02
+:summary: Security Encryption commands.
+
+The Security Encryption Commands bundle enhances the ${branding} system console by allowing administrators and integrators to encrypt and decrypt values directly from the console.
+
+The `security:encrypt` command allows plain text to be encrypted.
+This is useful when displaying password fields in a GUI.
+
+Below is an example of the security:encrypt command used to encrypt the plain text "myPasswordToEncrypt".
+The output, `bR9mJpDVo8bTRwqGwIFxHJ5yFJzatKwjXjIo/8USWm8=`, is the encrypted value.
+
+[source%nowrap.java]
+----
+${ddf-branding-lowercase}@local>security:encrypt myPasswordToEncrypt
+
+bR9mJpDVo8bTRwqGwIFxHJ5yFJzatKwjXjIo/8USWm8=
+----
+
+===== Installing the Security Encryption Commands
+
+This bundle is installed by default with the `security-encryption` feature.
+This bundle is tied specifically to the ${branding} console and can be uninstalled without causing any issues to other applications.
```

Comment:
"any issues to other applications" --> "issues for other applications"

Reply:
Actually, I dropped the `causing issues` clause altogether because the presumption should be that it won't cause issues and if it does, that's when we'll comment on it.